knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml

```diff
@@ -85,7 +85,7 @@ data:
             },
             "targets": [
                {
-                 "expr": "sum(knative_up{namespace=\"$namespace\", service=\"knative-openshift-metrics\", type=\"serving_status\"})",
+                 "expr": "floor(sum(knative_up{namespace=\"$namespace\", service=\"knative-openshift-metrics\"})/2)",
```

Comment:
This is correct because dividing by 2 gives a decimal number:
![image](https://user-images.githubusercontent.com/7945591/96986494-614a4880-152a-11eb-8f54-e1fe706c38ba.png)